nifi-nar-bundles/nifi-cybersecurity-bundle/nifi-cybersecurity-processors/src/main/java/org/apache/nifi/processors/cybersecurity/FuzzyHashContent.java

```diff
@@ -178,11 +161,11 @@ public void process(final InputStream in) throws IOException {
                     try (ByteArrayOutputStream holder = new ByteArrayOutputStream()) {
                         StreamUtils.copy(in,holder);
 
-                        if (context.getProperty(HASH_ALGORITHM).getValue().equals(allowableValueSSDEEP.getValue())) {
+                        if (algorithm.equals(allowableValueSSDEEP.getValue())) {
```

Comment:
added to the same class as the checkMinimumRequirements. wonder if we should create a separate class just to hold those help functions?